default.nix

```diff
@@ -87,6 +87,11 @@ let
         inherit pkgs;
         filter = localLib.isPlutus;
       };
+      # When building we want the git sha available in the Haskell code, previously we did this with
+      # a template haskell function that ran a git command however the git directory is not available
+      # to the derivation so this fails. What we do now is create a derivation that overrides a magic
+      # Haskell module with the git sha, this comes from `declInput.rev` if available (Hydra), otherwise
+      # it is read from the .git directory which is avilable on local builds.
```

Comment:
Does this mean we will never get a Hydra cache hit when building locally because the local version won't have that argument?

Reply:
I think so (lack of CAS strikes again) however this only affects the playground server and meadow so it shouldn't prove too problematic to anyone.